RxMarbles/OperatorCode.swift

```diff
@@ -135,30 +135,30 @@ extension Operator {
     func higlightedCode() -> NSAttributedString {
         let font = Font.code(.monoRegular, size:18)
         let src = NSMutableAttributedString(string: code, attributes: [
-            NSAttributedStringKey.font: font,
-            NSAttributedStringKey.foregroundColor: Color.codeDefault
+            NSAttributedString.Key.font: font,
```

Comment:
```suggestion
            .font: font,
```

and everywhere else in this file if you can